content/300-guides/050-database/870-using-prisma-with-mongodb.mdx

```diff
@@ -0,0 +1,326 @@
+---
+title: 'Using Prisma with MongoDB'
+metaTitle: 'Using Prisma with MongoDB'
+metaDescription: 'Guide to using Prisma with MongoDB'
+tocDepth: 2
+toc: true
+---
+
+<TopBlock>
+
+This guide discusses the concepts behind using Prisma and MongoDB, explains the commonalities and differences between MongoDB and other database providers, and leads you through the process for configuring your application to integrate with MongoDB using Prisma.
+
+</TopBlock>
+
+## What is MongoDB?
```

Comment:
Your wish is my command 🧞 

Mostly just taking the opportunity to throw a `float: right` in there 😌

Reply:
Old school 🧓 

But unfortunately Gatsy or something does weird things to it:
```
<img src="/docs/static/a4546ec668a59cd396325fff01e316d9/mongodb-logo.svg" style="css-float:right" width="195">
```

Reply:
Would look _great_ though: 
<img width="625" alt="image" src="https://user-images.githubusercontent.com/183673/161639259-48e29832-f716-45a6-a217-3662ffed5a07.png">

Reply:
The positioning of this that I'm seeing on the preview currently looks very strange to me: 

![image](https://user-images.githubusercontent.com/3910386/161705107-5ff4989f-2cf0-482d-9b65-a9b55abcd9f6.png)

Also we don't normally have logos in these, can you explain why this is being added?

Reply:
The positioning is off, as the code that Matt added is mangled by Gatsby somehow and does not work. I emulated what he wanted to achieve manually by fixing the code in the developer console of Chrome.

Matt added this, as I had left a comment here that this would be "neat". 
I think it makes the page look much nicer and also visually able to connect back to MongoDB.

Reply:
Removed logo for now, will include again once positioning is sorted.